express-static/public/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Static Middleware</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <h1>Static Middleware!</h1>
+    <img src="images/51dmUVNirNL._AC_SX679_.jpg" alt="gray hoodie" />
```

Comment:
These src URLs need to start with `/` or they may end up with the wrong path. If a URL does not start with `/` then the browser preprends the current URL onto it. So if you are on a page that is not at the root, the path will be wrong and Express static won't be able to locate the file.